paasta_tools/tron_tools.py

```diff
@@ -617,7 +644,14 @@ def format_tron_action_dict(action_config: TronActionConfig, use_k8s: bool = Fal
         # for Tron-on-K8s, we want to ship tronjob output through logspout
         # such that this output eventually makes it into our per-instance
         # log streams automatically
-        result["env"]["ENABLE_PER_INSTANCE_LOGSPOUT"] = "1"
+        # however, we're missing infrastructure in the superregion where we run spark jobs (and
+        # some normal tron jobs), so we take a slightly different approach here
+        if _use_suffixed_log_streams_k8s():
+            result["env"]["STREAM_SUFFIX_LOGSPOUT"] = (
+                "spark" if executor == "spark" else "tron"
+            )
+        else:
+            result["env"]["ENABLE_PER_INSTANCE_LOGSPOUT"] = "1"
```

Comment:
Oh TIL this is already available ++

Reply:
the per-instance logspout stuff has been running for a while - its just the suffix stuff that's new (and I just need to actually deploy the new logspout version to enable it + i guess make tron look at it :p)